README.md

```diff
@@ -112,6 +112,17 @@ q2 = q1(return_content_type=["computational", "experimental"])
 
 list(q2)
 ```
+### Obtaining only Result Counts 
+
+If only the number of queries is desired, the count function can be used. This query returns the number of experimental models associated with "hemoglobin".
+```python
+from rcsbsearchapi.search import TextQuery
+
+q1 = TextQuery("hemoglobin")
+
+# similar to exec, return_type and return_content_type can also be specified
```

Comment:
Good point, I forgot about this. In this case, I'd rephrase it as follows, and then move this section to _after_ the next section, "Return Types and Attribute Search":
```suggestion
# N.B., Just as shown above for `query()`, `return_type` and `return_content_type` can also be specified as parameters to `count()`
```